lib/task/sfBaseTask.class.php

```diff
@@ -17,6 +17,13 @@
  */
 abstract class sfBaseTask extends sfCommandApplicationTask
 {
+
+    protected $version;
```

Comment:
I don't see anyone using the version field. What's the reason you added this?

Reply:
Oh yes. Sorry, looks like $version is something custom i added long time ago to some tasks. Removed, no deprecated messages anymore.